blocks/eda-frontend/src/components/SchematicEditor/ComponentProperties.js

```diff
@@ -225,15 +305,19 @@ export default function ComponentProperties () {
             const rootKeyName = keyName.substr(0, 4)
             const typeId = rootKeyName + '_type'
             const helpId = rootKeyName + '_help'
-            if (compProperties !== undefined && compProperties[rootKeyName] !== null && compProperties[typeId] !== null) {
+            const compType = compProperties && compProperties[typeId]
+            const compHelp = compProperties && compProperties[helpId]
+            const helperText = errorFields[keyName]
+              ? getHelperText(compType)
+              : compHelp
+            if (compProperties && compProperties[rootKeyName] !== null && compType !== null && val[keyName] !== null) {
```

Comment:
```suggestion
            if (compProperties && compProperties[rootKeyName] !== null && compType !== null) {
```
remove null check for val here